### PR TITLE
GH-92 - Update Adobe Fonts to CSS Integration

### PR DESCRIPTION
**Pull request by auniverseaway**
* Moved to CSS-based AF integration
* Moved kit to adobe.com managed ID
* Added Adobe Clean & Adobe Clean Serif

Resolves: GH-92

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -33,14 +33,5 @@
...
-    var config = {
-            // Todo: pull in the kit Id from the ACF field in the admin
-            kitId: 'lkg3fdt',
-            scriptTimeout: 3000,
-            async: true
-        },
-        h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
-})(document);
 </script>
+<link rel="stylesheet" href="https://use.typekit.net/pnv6nym.css"/>
```
Comment (rofe) [inline head.html]:
```suggestion
<link rel="stylesheet" href="/hlx_fonts/pnv6nym.css"/>
```
@auniverseaway Thanks for the hint to the optimized kit. This is great, but for performance reasons, we should still use the `/hlx_fonts` wrapping @trieloff suggested in #92 - it allows us to reduce the number of 3rd party connections and deliver everything from the same domain.

Comment (rofe) [reply head.html]: Also, let's move the link tag to the others at the top of the document.

**Comment by auniverseaway** [comment]
@rofe Updated. However, on localhost, I get a 404. How were you able to successfully test this?

Screen Shot 2020-02-07 at 10 40 14 AM

**Comment by rofe** [comment]
Works for me™ 😄 

You may need to update to a newer CLI and/or have it rebuild the `.hlx` folder during `hlx up`.

**Comment by auniverseaway** [comment]
@rofe I did both already. If this works for you, please accept the change.

**Pull request merged**